Makefile

```diff
@@ -10,53 +10,60 @@ endif
 
 .PHONY: install
 install: ## make install # Install roles dependencies
-	@ansible-galaxy install --roles-path vendor/ -r requirements.yml
```

Comment:
I reckon' it is not the best thing to have hard coded the `vendor/` directory here. However the goal is to install external dependencies only.

Usually the `roles/`  directory is kept to define a custom set of roles within your ansible project. So I think we should not change this behaviour:

- `roles/` directory contains roles defined in the current project
- `vendor/`  directory contains **not commited** community modules

Could you keep `vendor/` or let me know what do you think please?